vsgenstubs4/_vapoursynth.part.pyi

```diff
@@ -311,6 +311,8 @@ class VideoFormat:
     subsampling_h: int
     num_planes: int
 
+    def __int__(self) -> int: ...
```

Comment:
Isn't this in the realm of too obvious things like __str__? You generally don't call these directly in scripts

Reply:
All classes have a `__str__` method because when you call `str(some_object)` it search for  `__str__` and then  `__repr__` otherwise. The reason why I added `__str__` is that it's a custom formatting string instead of the basic memory addressing.

I agree it's not that useful anyway but for this one, not all classes have the `__int__` method. Now you could do this explicitely without to look at the cython code:
```py
clip = clip.resize.Bicubic(format=clip2.format.id)
clip = clip.resize.Bicubic(format=int(out.format))
```